src/app/components/WorkDetail/DefinitionList.jsx

```diff
@@ -2,43 +2,42 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Html5Entities } from 'html-entities';
 import { Link } from 'react-router';
-import { isEmpty as _isEmpty } from 'underscore';
-import EBookList from '../List/EBookList';
+import { isEmpty as _isEmpty, isArray as _isArray } from 'underscore';
+import AuthorsList from '../List/AuthorsList';
 
 const htmlEntities = new Html5Entities();
-const elements = ['title', 'agents', 'instances', 'subjects', 'language'];
-const sorting = ['title', 'language', 'agents', 'subjects', 'instances'];
+
 export const labels = {
-  title: 'Title',
-  agents: 'Author, Creator, et al',
-  instances: 'Items',
+  // also provides sort
+  summary: 'Summary',
+  series: 'Series',
+  agents: 'Author',
+  subjects: 'Subject',
+  date_created: 'Desta Created',
+  issued_display: 'Date Issued',
   language: 'Language',
-  subjects: 'Subjects',
+  rights: 'Rights',
+  measurements: 'Measurements',
+  identifiers: 'Identifiers',
 };
+const elements = Object.keys(labels);
 
 /**
  * Build a definition list of elements from a bibliographic record provided
  * by Elastisearch.
  *
  * @param {object} props
  */
-export const DefinitionList = ({ eReaderUrl, data }) => {
-  data.sort((a, b) => sorting.indexOf(a[0]) - sorting.indexOf(b[0]));
-  const getIdentifier = author => (author.viaf && 'viaf') || (author.lcnaf && 'lcnaf') || 'name';
-  const linkToAuthor = author => ({
-    q: author[getIdentifier(author)],
-    field: getIdentifier(author),
-    showQuery: `"${author.name}"`,
-    showField: 'author',
-  });
-  const birthDate = entity => (
-    <span>
-      {(entity.birth_date_display || entity.death_date_display) && <span> (</span>}
-      {entity.birth_date_display && <span>{entity.birth_date_display}</span>}
-      {entity.death_date_display && <span> -- {entity.death_date_display}</span>}
-      {(entity.birth_date_display || entity.death_date_display) && <span>) </span>}
-    </span>
-  );
+export const DefinitionList = ({ work }) => {
+  /**
+   * Convert JSON object to array for parsing detail elements into
+   * a definition list for display.
+   *
+   * @param {object} work
+   * @return {string|null}
+   */
+  const workDetailsObject = workObj => Object.keys(workObj).map(key => [key, workObj[key]]);
```

Comment:
Try using `entries` instead of `map`, it'll give you what you want.

Reply:
The build was broken because of this, I changed it back.